tests/test_sshfsmount.cpp

```diff
@@ -152,6 +186,32 @@ struct SshfsMount : public mp::test::SftpServerTest
         return channel_read;
     }
 
+    // Check that a given command is invoked and that it throws when failing.
+    void test_failed_invocation(const std::string& fail_cmd)
+    {
+        CommandVector commands = {
+            {"echo $PWD/target", "/home/ubuntu/target\n"},
+            {"sudo /bin/bash -c 'P=/home/ubuntu/target; while [ ! -d $P/ ]; do P=${P%/*}; done; echo $P/'",
+             "/home/ubuntu/\n"},
+            {"id -u", "1000\n"},
+            {"id -g", "1000\n"}};
```

Comment:
Shouldn't these go into `default_cmds`? In this test, you don't want to verify that the command ordering is correct, but rather that the failure of `fail_cmd` is bubbled up correctly.

Reply:
Right.